internal/setup/flux/README.md

```diff
@@ -0,0 +1,2 @@
+**For Internal use only**
```

Comment:
I don't think we need to have an internal only section. If it's truly internal then it should live elsewhere. Or, we can make it generic enough to keep included.

Reply:
Ok. It's actually generic. I'll move it back to setup/flux.
The thing is noone except GitOps connector developers is supposed to use it. I'll add a sentence about that